src/main/kotlin/Square.kt

```diff
@@ -0,0 +1,3 @@
+class Square(var color: Int) {
```

Comment:
Does the square really change its color? I think, it should be a `val`.

Reply:
Fixed